sites/all/themes/wille/template.php

```diff
@@ -112,22 +112,23 @@ function wille_ting_collection_view_alter(&$build) {
  * Wraps the cover in a link to the material.
  */
 function wille_ting_object_cover($variables) {
-  $ding_entity_id = $variables['elements']['#object']->ding_entity_id;
-  $type = $variables['meta_for_labels']['type'];
-  $quota_explanation = t('This material is a !type and is not on your quota', ['!type' => $type]);
-  if (isset($variables['meta_for_labels']['on_quota']) && $variables['meta_for_labels']['on_quota']) {
-    $quota_explanation = t('This material is a !type and is on your quota', ['!type' => $type]);
-  }
-  $attributes['aria-label'] = $quota_explanation;
   $attributes = array(
     'class' => implode(' ', $variables['classes']),
   );
+  if (isset($variables['label_info']) && isset($variables['label_info']['type']) && isset($variables['label_info']['on_quota'])) {
+    $type = $variables['label_info']['type'];
+    $quota_explanation = t('This material is a !type and is not on your quota', ['!type' => $type]);
+    if (isset($variables['label_info']['on_quota']) && $variables['label_info']['on_quota']) {
```

Comment:
You don't need to check if `$variables['label_info']['on_quota']` is set here. You already know that it is set (cf. line 118).

Consider rewriting to
```
$quota_explanation = $variables['label_info']['on_quota']
                   ? t('This material is a !type and is on your quota', ['!type' => $type])
                   : t('This material is a !type and is not on your quota', ['!type' => $type]);
```